time stamp
uint8 heart_rate
float32[] rr_intervals